src/protocol/command.rs

```diff
@@ -42,124 +43,6 @@ impl Command {
     }
 
     pub fn new(string: &str) -> R<Command> {
-        use std::collections::VecDeque;
```

Comment:
we start by outlining the `Parser` struct and placing it into `src/protocol/parser.rs`.